app/ui/generalpage.ui

```diff
@@ -458,14 +458,17 @@
               <height>16777215</height>
              </size>
             </property>
+            <property name="suffix">
+             <string>MB</string>
+            </property>
             <property name="minimum">
-             <number>11</number>
+             <number>100</number>
             </property>
             <property name="maximum">
-             <number>1500</number>
+             <number>16000</number>
```

Comment:
The max in activeframepool does not match this. It is 16 * 1024, not 16000.

Reply:
will update